vfs/vfstest/fs.go

```diff
@@ -40,79 +45,112 @@ const (
 //
 // If useVFS is not set then it runs the mount in a subprocess in
 // order to avoid kernel deadlocks.
-func RunTests(t *testing.T, useVFS bool, minimumRequiredCacheMode vfscommon.CacheMode, enableCacheTests bool, mountFn mountlib.MountFn) {
+//
```

Comment:
As far as I can see all the changes in this file at this point and after are spurious - can you revert them?